### ISSUE TITLE
Add Cypress test for SwipeableModal.tsx (for swipe gestures)

### ISSUE THREAD

**Issue by Aprillion** [OpenSourceRaidGuild/atom-pwa#82]
https://github.com/OpenSourceRaidGuild/atom-pwa/tree/next/src/components/shared/swipeable-modal/SwipeableModal.tsx

**Comment by Aprillion** [comment]
Actually, it's covered as good as possible by Jest integration tests.

The uncovered method `onPan(event: HammerInput)` is better tested by an E2E test (...I think 🤔). But I am afraid that figuring out how to test swiping in Cypress is too much for my time-budget today => this issue is free for grabbing.

**Comment by MohitPopli** [comment]
> Actually, it's covered as good as possible by Jest integration tests.
> 
> The uncovered method `onPan(event: HammerInput)` is better tested by an E2E test. I an afraid that figuring out how to test swiping in Cypress is too much for my time-budget today => this issue is free for grabbing.

I gave it a try just now but got no luck with cypress and hammer `pan` event. Though I have verified the order of events by debugging events in this example https://codepen.io/jtangelder/pen/ABFnd

Order of events are
`PointerMove`
`PointerDown` ==> tap detected
`PointerMove` ==> right/left/up/down gesture
`PointerUp`

Triggering same event flow in cypress didn't swipe modal. Not sure if I missed something. Curious to see if someone else get this working :)
